scripts/install-windows-prereqs.ps1

```diff
@@ -582,17 +602,29 @@ function Install-DCAP-Dependencies {
                     Throw "Failed searching for $driver driver"
                 }
                 $output | ForEach-Object {
-                    if($_.Contains($drivers[${OS_VERSION}][$driver]['description'])) {
+                    if(($_.Contains($drivers[${OS_VERSION}][$driver]['description'])) -and ($OS_VERSION -eq 'WinServer2016')) {
                         Write-Output "Removing driver $($drivers[${OS_VERSION}][$driver]['location'])"
                         Remove-DCAPDriver -Name $drivers[${OS_VERSION}][$driver]['location']
                     }
                 }
-                Write-Output "Installing driver $($drivers[${OS_VERSION}][$driver]['location'])"
-                $install = & $devConBinaryPath install "$($inf.FullName)" $drivers[${OS_VERSION}][$driver]['location']
-                if($LASTEXITCODE) {
-                    Throw "Failed to install $driver driver"
+                try
+                {
+                    Write-Output "Installing driver $($drivers[${OS_VERSION}][$driver]['location'])"
+                    $install = & $devConBinaryPath install "$($inf.FullName)" $drivers[${OS_VERSION}][$driver]['location']
+                    Write-Output $install
+                    if($LASTEXITCODE) {
+                        Throw "Failed to install $driver driver"
```

Comment:
Can you do a specific check for the error code for when the driver is already installed and fall through to trying an install instead of using exception handling?